browser/notifications/brave_notification_platform_bridge_mac.mm

```diff
@@ -0,0 +1,113 @@
+/* Copyright (c) 2019 The Brave Software Team. Distributed under the MPL2
+ * license. This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#include "brave/browser/notifications/brave_notification_platform_bridge_mac.h"
+#include "brave/browser/notifications/notification_platform_bridge_notification_timeout_mac.h"  // NOLINT
+#include "brave/browser/notifications/brave_alert_dispatcher_mac.h"
+#include "base/strings/sys_string_conversions.h"
+#include "base/mac/scoped_nsobject.h"
+
+// static
+std::unique_ptr<NotificationPlatformBridge>
+NotificationPlatformBridge::Create() {
+  base::scoped_nsobject<BraveAlertDispatcher> alert_dispatcher(
+      [[BraveAlertDispatcher alloc] init]);
+  return std::make_unique<BraveNotificationPlatformBridgeMac>(
+      [NSUserNotificationCenter defaultUserNotificationCenter],
+      alert_dispatcher.get());
+}
+
+BraveNotificationPlatformBridgeMac::BraveNotificationPlatformBridgeMac(
+    NSUserNotificationCenter* notification_center,
+    id<AlertDispatcher> alert_dispatcher)
+    : NotificationPlatformBridgeMac(notification_center, alert_dispatcher) {
+  notification_platform_bridge_notification_timeout_ =
+      [[[NotificationPlatformBridgeNotificationTimeoutMac alloc] init] retain];
+}
+
+BraveNotificationPlatformBridgeMac::~BraveNotificationPlatformBridgeMac() {
+  [notification_platform_bridge_notification_timeout_ release];
+  notification_platform_bridge_notification_timeout_ = nil;
+}
+
+void BraveNotificationPlatformBridgeMac::Display(
+    NotificationHandler::Type notification_type,
+    Profile* profile,
+    const message_center::Notification& notification,
+    std::unique_ptr<NotificationCommon::Metadata> metadata) {
+  base::scoped_nsobject<NotificationBuilder> builder(
```

Comment:
we don't want to copy the entire method, just call the superclass and then this https://github.com/brave/brave-core/pull/3850/files#diff-a8d2c73dbfa104437b132a5ca263249bR110
I think you'll have to call `NSUserNotification* toast = [builder buildUserNotification];` again, but that's fine

Reply:
Ah yes did not push change as was unable to build due to Xcode upgrade, will push change soon

Reply:
Fixed